x/fee/client/cli/query.go

```diff
@@ -71,12 +74,18 @@ func GetFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			symbol := args[0]
 
-			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "fee", fee.QueryFungibleTokenFeeSetting, symbol), nil)
+			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "token", fungible.QueryTokenData, symbol), nil)
 			if err != nil {
 				fmt.Printf("Could not get fungible token fee setting: %s\n", err)
```

Comment:
@githubckgoh1439 fix the error message.

Reply:
Fixed the message using 'fungible token not exist'.